Assets/Infiltration/Scripts/AI/Checks/CheckEnemyInFOVRange.cs

```diff
@@ -0,0 +1,54 @@
+using BehaviorTree;
+using UnityEngine;
+
+namespace Infiltration
+{
+    public class CheckEnemyInFOVRange : Node
+    {
+        private readonly Transform _transform;
+        private const int PlayerLayerMask = 1 << 6;
+
+        private readonly SpriteRenderer _renderer;
+        private readonly float _fovRange;
+        public CheckEnemyInFOVRange(Transform transform, SpriteRenderer renderer, float fovRange)
```

Comment:
Consider implementing a constructor without arguments and using public properties tagged with the ```ExposedInVisualEditor``` attribute (which will be added later to the codebase) with ```set``` or ```init``` accessors.

Reply:
Empty constructor already existing in `Node`, waiting for the `ExposedInVisualEditor `